pom.xml

```diff
@@ -131,6 +131,7 @@
     <version.jackson.databind>2.10.1</version.jackson.databind>
     <version.org.jboss.resteasy>3.6.1.SP5</version.org.jboss.resteasy>
     <version.hibernate>5.3.10.Final</version.hibernate>
+    <version.org.infinispan>9.3.6.Final</version.org.infinispan>
```

Comment:
Shouldn't we use Infinispan version bundled with EAP?

Reply:
Yes, that is only required for the tests in the model module which use a RESOURCE_LOCAL persistence unit, and that seems to need an additional artifact. Should work out of the box for the standard persistence unit.

Reply:
Isn't it enough to use the version of infinispan defined in EAP BOM?

Reply:
that dependency is not in the EAP BOM

Reply:
I found it there. I think it should be removed. See: https://mvnrepository.com/artifact/org.jboss.bom/eap-runtime-artifacts/7.2.6.GA
and
https://maven.repository.redhat.com/ga/org/jboss/bom/eap-runtime-artifacts/7.2.6.GA/eap-runtime-artifacts-7.2.6.GA.pom

Reply:
I'd add the runtime BOM.

Reply:
I didn't realize we don't use the runtime-artifacts BOM. Not sure it is worth adding it just for this one artifact for tests. 
Let's leave it as is